codegen/src/main/scala/akka/grpc/gen/javadsl/Method.scala

```diff
@@ -50,13 +54,23 @@ final case class Method(
 
 object Method {
   def apply(descriptor: MethodDescriptor): Method = {
+
+    val comment = {
+      // Use ScalaPB's pimp classes to avoid replicating the logic for comment extraction
+      // Note that this be problematic if/when ScalaPB uses scala-specific stuff to do that
+      implicit val ops = new DescriptorImplicits(GeneratorParams(), descriptor.getFile.getDependencies.asScala :+ descriptor.getFile)
```

Comment:
Ah, you create a new empty set of parameters here instead of passing them through from the plugin. For determining the comment that's sufficient, so since this is so self-contained and documented this seems fine!